cmd/crates/soroban-spec-typescript/src/project_template/src/invoke.ts

```diff
@@ -17,6 +17,7 @@ import type {
 
 export type Tx = Transaction<Memo<MemoType>, Operation[]>;
 
+export class SendFailedError extends Error { }
```

Comment:
This error /the pattern of throwing the error is being added in https://github.com/stellar/soroban-tools/pull/1034 as well